ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -195,6 +195,10 @@
 {%             if peer_group.bfd is arista.avd.defined(true) %}
 | BFD | True |
 {%             endif %}
+{%             if peer_group.bfd_intervals is arista.avd.defined and peer_group.bfd_intervals.interval is arista.avd.defined and peer_group.bfd_intervals.min_rx is arista.avd.defined and peer_group.bfd_intervals.multiplier is arista.avd.defined %}
```

Comment:
You don't need the first check. The deeper tests will catch the parent dict correctly.